api/tests/prescription/campaign/integration/infrastructure/repositories/campaign-participant-activity-repository_test.js

```diff
@@ -72,6 +72,76 @@ describe('Integration | Repository | Campaign Participant activity', function ()
         expect(campaignParticipantsActivities).to.have.lengthOf(1);
         expect(campaignParticipantsActivities[0].participantExternalId).to.equal('The good');
       });
+
+      it('Returns the last shared shared participation', async function () {
```

Comment:
**suggestion**: enlever le shared en trop

Reply:
**suggestion** : mettre un nom plus compréhensible 
`Returns the most recent participation with the shared participation Id`